xgmutil/stream.py

```diff
@@ -1,26 +1,33 @@
 import jax.numpy as jnp 
 import jax.random as rnd
 import jax
-import sys
 import os
 
 class Stream:
     '''Stream'''
     def __init__(self, **kwargs):
 
         self.force_no_gpu = kwargs.get('force_no_gpu',False)
-        self.seed         = kwargs.get('seed',123456789)
```

Comment:
@1cosmologist can we keep this variable called seed in the initialization and allow for it to be either an integer (to retain the functionality before this change) or a key (as you have it now)?

Reply:
Ok, it can be added. But which takes precendence when both are provided?

Reply:
This has been resolved by using the type of `self.seedkey` to determine how it works.